app/services/induction/find_relevant_to.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class Induction::FindRelevantTo < BaseService
+  def call
+    query = participant_profile.induction_records
+
+    query = add_provider_to(query:) if lead_provider.present? || delivery_partner.present?
```

Comment:
I wonder if we could prioritise `end_date_null`/`active` records regardless of the `current_not_latest_record` generally?

Reply:
Could you explain a bit more how you'd like this to work?

Reply:
chatted in person but for posterity: when prioritising records, we could also add `end_date: nil` to the list. So first we look for any records where the `end_date` is `nil`, then ones with the latest start date, and then created at etc.